Easydict/objc/Configuration/EZConfiguration.m

```diff
@@ -362,19 +362,9 @@ - (void)setHideMenuBarIcon:(BOOL)hideMenuBarIcon {
     
     [NSUserDefaults mm_write:@(hideMenuBarIcon) forKey:kHideMenuBarIconKey];
     
-    if (!Configuration.shared.enableBetaNewApp) {
```

Comment:
I wonder if we could remove `EZConfiguration` entirely, it looks we already use `Configration` instead now.

Reply:
Yeah, I will remove `EZConfiguration` in the next PR because it has a lot of keys. I want to remove it carefully.

Reply:
OK